icon_validator/rules/plugin_validators/confidential_validator.py

```diff
@@ -33,8 +31,8 @@ def validate_emails(content: [str], path_to_file: str):
             for match in matches:
                 user_email_pattern = re.compile(r"user\d*\@example.com")
                 if not user_email_pattern.search(match):
-                    ConfidentialValidator.violations.append(f"{path_to_file}, line: {i + 1}")
-
+                    print(
+                        f"{YELLOW}WARNING: Email does not match recommended example user@example.com")
```

Comment:
Actually, I would still try to include the path to the file and line number just to make it easier for the user to work out where the issue is happening.
```suggestion
                        f"{YELLOW}WARNING: Email does not match recommended example user@example.com\n{path_to_file}, line: {i + 1}")
```

Reply:
ok no worries, just added it 🙂